services/meta/client.go

```diff
@@ -1251,6 +1258,14 @@ func (e errRedirect) Error() string {
 	return fmt.Sprintf("redirect to %s", e.host)
 }
 
+type errCommand struct {
```

Comment:
Is there a reason to make thing a struct instead of just an aliased type? `type errCommand string`